candle/my_timetable/my_timetable.py

```diff
@@ -22,11 +30,32 @@ def get_timetable(id_):
     id_ = int(id_)
     return current_user.timetables.filter(UserTimetable.id_ == id_).first_or_404()
 
+
+def get_highlighted_lesson_ids(user_timetable_id: int) -> Set[int]:
```

Comment:
Niekedy sa s tým ešte skúsim pohrať a zistiť ako sa to v sqlalchemy robí poriadne... (zatiaľ to môže zostať takto)